tools/sof-dump-status.py

```diff
@@ -98,7 +98,7 @@ def _loadACard(self):
         for idx in range(0, len(output), 2):
             card_info={}
             card_info['id'] = output[idx].strip().split(' ')[0]
-            card_info['short'] = output[idx].split(' ')[-1].strip()
+            card_info['short'] = output[idx].split(' - ')[-1].strip()
```

Comment:
A quick question here. is the spliter **space_dash_space** here? it looks very like to '-'
Please clarify. If there is an example is better.

Reply:
so what is different between those change? you find the short name with the 'space' ?

Reply:
@xiulipan @Bin-QA 
Yes, it is space_dash_space. Our SOF is ok for space only because we don't use any space in our short card name. But as the test case is targeted to be used widely, we should consider some corner cases. I found this bug when I try to run it on my legacy hda driver.